docs/machine-learning/how-to-guides/install-model-builder.md

```diff
@@ -40,25 +40,25 @@ ML.NET Model builder can be installed either through the Visual Studio Marketpla
 
 1. In the menu bar, select **Tools** > **Extensions and Updates**
 
-    ![VS201701](./media/install-model-builder/VS201701.png)
+    ![VS2017 open manage extensions dialog](./media/install-model-builder/VS201701.png)
```

Comment:
Oh sorry I wasn't clear. I meant the image filenames

Reply:
Also best practice is to not use upper case filenames with git

Reply:
#resolved